### PR TITLE
Add BikeShare mode

### PR DESCRIPTION
**Pull request by xlqian**
add bikeshare mode:
image

zoom in details
image

### CODE REVIEW

File: routing/js/valhalla.js [hunk 1]
```diff
@@ -999,6 +999,34 @@ app.controller('RouteController', function($scope, $rootScope, $sce, $http) {
       });
     }
 
+    if (document.getElementById('bikeshare_btn') != undefined) {
```
Comment (xlqian) [inline routing/js/valhalla.js:1004]: it's just a ctrl+V ctrl+C from other buttons

**Review by kevinkreiser — APPROVED** [review]

**Comment by kevinkreiser** [comment]
@xlqian im not sure how this was missed, is it still ready to merge? code looked fine to me but I didn't test it locally

**Comment by xlqian** [comment]
4 years later, finally someone noticed ths PR, 🤣, yes, i have been using this feature on my side for years, it's mergeable.

**Comment by xlqian** [comment]
:tada:

**Pull request merged**